hopp/simulation/technologies/dispatch/power_storage/simple_battery_dispatch_heuristic.py

```diff
@@ -73,22 +73,22 @@ def _set_power_fraction_limits(self, gen: list, grid_limit: list):
     @staticmethod
     def enforce_power_fraction_simple_bounds(power_fraction) -> float:
         """ Enforces simple bounds (0,1) for battery power fractions."""
```

Comment:
Maybe should change the docstring to 0.9 instead of 1 for the bounds. I think ideally bounds would be an assignable variable.

Reply:
I agree. I think this is part of a bigger problem where need more configurability at the dispatch level. A quick fix would be to add power fraction args to `HybridDispatchOptions`, but it might be worth a more serious look

Reply:
I agree. I think this is part of a bigger issue where we need more configurability for the dispatch, with varying config options for different strategies. A quick fix would be to add power fraction options to `HybridDispatchOptions`, but I think it deserves a bit more work than that